bindings/python/Cargo.toml

```diff
@@ -13,21 +13,20 @@ rayon = "1.3"
 serde = { version = "1.0", features = [ "rc", "derive" ]}
 serde_json = "1.0"
 libc = "0.2"
-env_logger = "0.7.1"
-pyo3 = "0.12"
-numpy = "0.12"
-ndarray = "0.13"
+env_logger = "0.9.0"
+pyo3 = "0.16.2"
+numpy = "0.16.2"
+ndarray = "0.15"
 onig = { version = "6.0", default-features = false }
-itertools = "0.9"
+itertools = "0.10"
```

Comment:
Since we're updating dependencies, upgrade `env_logger` , `ndarray` and `itertools` along the way.

Reply:
Reverted to only upgrade pyo3 and numpy.